components/export/ui/src/main/resources/PhenoTips/ExportFilter.xml

```diff
@@ -212,6 +212,38 @@
 #else
   #set ($query = '')
 #end
+
+## Rights management
```

Comment:
I don't like this approach, but to me it seems that using the internal security components which depend on ```Patient``` will be too slow.